lib/tasks/flag_processing.rake

```diff
@@ -2,30 +2,30 @@ namespace :db do
-    min_count = 3
-    threshold = 0.75
     # geometry
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("geometry", "yes", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("geometry", "no", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("geometry", "fix", min_count, threshold))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("geometry", "yes", 1))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("geometry", "no", 1))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("geometry", "fix", 1))
     # colors
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "pink", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "yellow", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "green", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "gray", min_count, threshold))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "pink", 1))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue", 1))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "yellow", 1))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "green", 1))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "gray", 1))
     # multicolor
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue,green", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "green,yellow", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue,pink", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue,yellow", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "gray,pink", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "pink,yellow", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "green,pink,yellow", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "green,pink", min_count, threshold))
-    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue,pink,yellow", min_count, threshold))
+    Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", "blue,green", 1))
```

Comment:
Rather than build these manually, which invites error, could there be a static method on Flag that returns distinct values for a given task so that you can do something like this?

```
Flag.distinct_task_values("color").each do |value|
   Flag.connection.execute(build_polygon_consensus_query_for_task_value("color", value, 1)) 
end
```